src/sentry/api/endpoints/project_group_index.py

```diff
@@ -99,6 +101,20 @@ def get(self, request: Request, project) -> Response:
             stats_period=stats_period,
         )
 
+        hashes = request.GET.getlist("hash", [])
+        if hashes:
```

Comment:
Do we want to limit how many hashes we support here?

Reply:
yeah, we can add a limit in. will be easy to change in the future if we ever want to increase/decrease the limit